src/EventStore.Core/Services/Storage/ReaderIndex/IndexCommitter.cs

```diff
@@ -80,12 +82,16 @@ public class IndexCommitter<TStreamId> : IndexCommitter, IIndexCommitter<TStream
 			_streamExistenceFilterInitializer = streamExistenceFilterInitializer;
 			_indexChk = indexChk;
 			_additionalCommitChecks = additionalCommitChecks;
+			_indexStatusTracker = indexStatusTracker;
```

Comment:
consider calling it `statusTracker` without `index`, index is clear enough from the context i think